vhdbuilder/packer/components.json

```diff
@@ -316,7 +302,7 @@
         "v1.7.0"
       ],
       "multiArchVersions": [
-        "v1.9.0"
```

Comment:
also here.

Reply:
azurefile-csi:v1.8.0 is the first version with 'multi-arch'. I will use 1.8.0 instead of 1.9.0. Current RP doesn't use 1.9.0 at all.